make/go.mk

```diff
@@ -4,19 +4,4 @@ GO_PACKAGE_REPO_NAME ?= $(shell basename $$PWD)
 GO_PACKAGE_PATH ?= github.com/${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}
 
 GO111MODULE?=on
-export GO111MODULE
-
-.PHONY: build
-## Build the operator
-build: $(OUT_DIR)/operator
-
-$(OUT_DIR)/operator:
-	$(Q)CGO_ENABLED=0 GOARCH=amd64 GOOS=linux \
-		go build ${V_FLAG} \
-		-ldflags "-X ${GO_PACKAGE_PATH}/cmd/manager.Commit=${GIT_COMMIT_ID} -X ${GO_PACKAGE_PATH}/cmd/manager.BuildTime=${BUILD_TIME}" \
-		-o $(OUT_DIR)/bin/toolchain-e2e \
-		cmd/manager/main.go
```

Comment:
Why did you remove the build target?

Reply:
ah, I removed it when I also removed the `cmd` package, but I forgot to restore it afterwards (when I realized that the test failed)

Reply:
done in 01010e8